platform/linux-generic/miniz/miniz_tdef.h

```diff
@@ -47,7 +47,8 @@ enum
 /*  flags: The max match finder probes (default is 128) logically OR'd against the above flags. Higher probes are slower but improve compression. */
 /* On return: */
 /*  Function returns a pointer to the compressed data, or NULL on failure. */
-/*  *pOut_len will be set to the compressed data's size, which could be larger than src_buf_len on uncompressible data. */
+/*  *pOut_len will be set to the compressed data's size, which could be */
+/*  larger than src_buf_len on incompressible data. */
```

Comment:
Miniz files (miniz_tdef.h and miniz_tinfl.h) are copies from open source. It's better to not modify those, so that it's easier to compare / upgrade their version in the future. Please remove edits from those two files.